tests/phpunit/tests/avatar.php

```diff
@@ -56,19 +56,27 @@ public function test_get_avatar_url_rating() {
 	}
 
 	/**
+	 * Ensure the get_avatar_url always returns an HTTPs scheme for gravatars.
```

Comment:
```suggestion
	 * Ensures the get_avatar_url always returns an HTTPS scheme for gravatars.
```
Additional nit: "HTTPS" and "https" are both used in the docs here. Pick one or I'll never speak to you again (or if that's a positive, I'll never stop speaking to you). Same for "http" vs "HTTP"

Reply:
Went with uppercase as it appears to be more-common in the docs rather than the lowercase version.